builtin/fstk/tabview.lua

```diff
@@ -138,11 +138,9 @@ end
 
 --------------------------------------------------------------------------------
 local function tab_header(self)
-
 	local toadd = ""
 
-	for i=1,#self.tablist,1 do
-
+	for i = 1, #self.tablist, 1 do
```

Comment:
You could remove all the `1`s in the numeric for loops btw., eg. here:
`for i = 1, #self.tablist do`
(It doesn't really matter though.)